spec/features/restaurant_spec.rb

```diff
@@ -54,10 +53,10 @@
     expect(page).to have_content("Zip code can't be blank")
   end
 
-  scenario 'user clicks restaurant show link' do
+  scenario 'user clicks restaurant link' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.